An update of the recent data "Hydroxychloroquine in patients with COV… (#247)
greenelab/covid19-review: content/20.treatments.md

For all of these 15 identifiers, no data uploads or links to publications could be located on the pre-registrations.
At the very least, an inability to locate primary data means such claims cannot be verified.
Additionally, citation of unavailable sources cast doubt on policies and research based on the assumption that HCQ is effective in treating COVID-19.
A randomized, open label, non-placebo, trial of 150 COVID-19 patients was conducted parallelly in 16 government designated COVID-19 centers in China to study safety and efficacy of HCQ[@doi.org/10.1101/2020.04.10.20060558].
The trial studied efficacy of HCQ in conjunction with standard of care (SOC) vs only SOC in 150 infected patients assigned randomly to two respective groups (75 each group). 

> Most of the cases (98%) in this cohort were mild to moderate with average age of patients being 46 years.

Reviewer: Revise to, Most of the COVID-19 cases (98%)

Author: The change is made

> Most of the SARS-CoV-2 cases (98%) in this cohort were mild to moderate with average age of patients being 46 years.

Reviewer 2: ```suggestion
Most of the SARS-CoV-2 cases in this cohort were mild to moderate (98%), and the average age of the patients was 46 years.
```

The primary end point of the study was the negative conversion rate of SARS-CoV-2 in 28 days and the investigators found no difference in this parameter between the groups. 

> The secondary end points were amelioration of the symptoms of the disease. 

Author: The symptoms are added

The median time of symptoms alleviation in both cases was similar (19 days in HCQ+SOC vs 21 days in SOC). 
However, an interesting aspect of the analyses by the investigators showed that, HCQ was efficacious in alleviation of symptoms in patients receiving no antiviral treatment in post hoc analysis. 
Additionally, there was an increase in lymphocytes in SOC+HCQ group as opposed to only SOC group at a borderline p value (p=0.57). Following this, the authors hypothesized that addition of HCQ to the current SOC could decrease inflammatory response and subsequently prevent multiorgan failure and death. 
Additionally, one of the key results of this trial was that the 30% of the patients that received SOC+ HCQ had adverse outcomes as opposed to 8.8% in only SOC. The most adverse outcomes being diarrhea in SOC+HCQ group(10%). 
The study has few limitations. One of the limitations of the study was that the cohort mostly consisted of patients with mild/moderate disease symptoms with an average age group 46 and did not include aged group known to be at high risk category of the disease. 

> Although, authors claimed that HCQ could have beneficial effect hindering disease progression, multiorgan failure and death in COVID-19, that is yet to be tested for severe cases.

Author: The data they presented in the paper is based on mild/moderate symptoms as they represent 98% of the cohort population. However, the authors claimed that multiorgan failure and death can be prevented by HCQ  administration although that is the manifestion of severe disease symptoms. This hypothesis is needed to be tested.

> Indeed, larger population needs to be tested to validate the increase of lymphocytes in SOC+HCQ as although, the result was promising, it lacked statistical significance.  Additionally, SOC used in this study included the use of antivirals which could provide confounding effects. Thus, to better understand the drug effects of HCQ, a better study design needs to be outlined, where SOC should not involve the use of anti-virals. 

Author: Lopinavir-Ritonavir, Arbidol, Oseltamivir, Virazole, Entecavir, Ganciclovir and/or Interferon-alpha.
This is now added in text.

> Indeed, larger population needs to be tested to validate the increase of lymphocytes in SOC+HCQ as although, the result was promising, it lacked statistical significance.  Additionally, SOC used in this study included the use of antivirals (Lopinavir-Ritonavir, Arbidol, Oseltamivir, Virazole, Entecavir, Ganciclovir and/or Interferon-alpha) which could provide confounding effects. Thus, to better understand the drug effects of HCQ, a better study design needs to be outlined, where SOC should not involve the use of anti-virals. 

Reviewer 2: ```suggestion  
Indeed, a larger sample needs to be tested to validate whether lymphocyte counts do increase in patients treated with SOC+HCQ; although the result in the present analysis appeared promising, it lacked statistical significance.
Additionally, in this study, SOC included the use of antivirals (Lopinavir-Ritonavir, Arbidol, Oseltamivir, Virazole, Entecavir, Ganciclovir and/or Interferon-alpha), which appeared to introduce confounding effects. 
Thus, to better assess the effects of HCQ, an alternate study design needs to be outlined where SOC does not involve the use of anti-virals.
```

In this trial, the samples collected from the upper respiratory tract introduced false negatives and thus finding biomarker for the disease is incumbent. Future studies to unravel serum/ urinary/ Bronchoalveolar lavage (BAL) markers for the disease is  warranted.
Another limitation of the study was that the HCQ treatment was provided at 16 day delay from the symptom onset.  A quicker treatment regimen with HCQ needs to be investigated for early disease management. Overall, the study holds promising data albeit needs validation in independent population cohort. Also from safety point of view, the adverse effect in manageable.
##### Summary

> In vitro evidence shows HCQ may be an effective therapeutic against SARS-CoV-2 and COVID-19. 

Reviewer: _In vitro_ evidence shows Choloroquine and Remdesivir may be an effective therapeutic against SARS-CoV-2  (doi:10.1038/s41422-020-0282-0). Using African green monkey kidney cells and human liver cancer cells, the study evaluated the antiviral efficiency of 5 FDA-approved (ribavirin, penciclovir, nitazoxanide, nafamostat, choloroquine, remdesivir, and favipiravir)  drugs against a clinical isolate of 2019-nCoV _in vitro_. The authors quantified viral copy numbers in cell supernatant and confirmed the results using viral nucleoprotein expression through immunofluorescence. Both Choloroquine and Remdesivir blocked virus infection at low-micromolar concentration and showed a high selectivity index (SI). This study investigated basic mechanisms using African green monkey kidney cells and human liver cancer cells. Including a lung cell type might have been more applicable to use a lung cell type, since we know that the lungs are what is affected by SARS-CoV-2. Further studies in vivo are required to study the effects of Choloroquine and Remdesivir.

Author: Looks great.

For all of these 15 identifiers, no data uploads or links to publications could be located on the pre-registrations.
At the very least, an inability to locate primary data means such claims cannot be verified.
Additionally, citation of unavailable sources cast doubt on policies and research based on the assumption that HCQ is effective in treating COVID-19.

> A randomized, open label, non-placebo, trial of 150 COVID-19 patients was conducted in parallel in 16 government-designated COVID-19 centers in China to assess the safety and efficacy of HCQ [@doi:10.1101/2020.04.10.20060558].

Reviewer 3: ```suggestion
A randomized, open-label, non-placebo trial of 150 COVID-19 patients was conducted in parallel in 16 government-designated COVID-19 centers in China to assess the safety and efficacy of HCQ [@doi:10.1101/2020.04.10.20060558].
```

The trial examined the efficacy of HCQ in conjunction with standard of care (SOC) compared to SOC alone in 150 infected patients assigned randomly to the two respective groups (75 per group). 
Most of the SARS-CoV-2 cases in this cohort were mild to moderate (98%), and the average age of the patients was 46 years.

> The primary end point of the study was the negative conversion rate of SARS-CoV-2 in 28 days, and the investigators found no difference in this parameter between the groups. 

Reviewer 3: ```suggestion
The primary endpoint of the study was the negative conversion rate of SARS-CoV-2 in 28 days, and the investigators found no difference in this parameter between the groups.
```

> The secondary end points were amelioration of the symptoms of the disease such as axillary temperature ≤36.6°C,SpO2 >94% on room air,and disappearance of symptoms like shortnes of breath, cough, sore throat.

Reviewer 3: ```suggestion
The secondary endpoints were an amelioration of the symptoms of the disease such as axillary temperature ≤36.6°C, SpO2 >94% on room air, and disappearance of symptoms like shortness of breath, cough, and sore throat.
```

> The median time to symptom alleviation was similar across conditions (19 days in HCQ+SOC vs 21 days in SOC). 

Reviewer 3: ```suggestion
The median time to symptom alleviation was similar across different conditions (19 days in HCQ+SOC vs. 21 days in SOC).
```

Reviewer 2: ```suggestion
The median time to symptom alleviation was similar across the two conditions (19 days in HCQ+SOC vs 21 days in SOC). 
```
Building on @matfax suggestion!

> However, the investigators reported an interesting finding revealed in _post hoc_ analysis: controlling for the administration of anti-virals yielded an effect of HCQ on the alleviation of symptoms, with a reported hazard ratio of 8.83.

Reviewer 3: ```suggestion
However, the investigators reported an interesting finding revealed in _post hoc_ analysis: controlling for the administration of antivirals yielded an effect of HCQ on the alleviation of symptoms, with a reported hazard ratio of 8.83.
```

> However, the 95% confidence interval of 1.09 to 71.3 suggests that this analysis may be under-powered and should be interpreted cautiously, as only 28 patients total (14 in each group) met this criterion.

Reviewer 3: ```suggestion
However, the 95% confidence interval of 1.09 to 71.3 suggests that this analysis may be underpowered and should be interpreted cautiously, as only 28 patients total (14 in each group) met this criterion.
```

Additionally, there was a non-significant trend towards a greater number of lymphocytes in the SOC+HCQ group compared to the SOC alone group (mean of 0.062 versus 0.008; p=0.57). 
Given the improvement in CRP levels and the possible improvement in lymphocyte count, the authors hypothesized that the addition of HCQ to the current SOC could decrease the inflammatory response and subsequently prevent multiorgan failure and death. 
Additionally, one of the key results of this trial was that the 30% of the patients receiving SOC+HCQ reported adverse outcomes as opposed to 8.8% of patients receiving only SOC. 

> The most common adverse outcome in the SOC+HCQ group was diarrhea (10% vs 0% in the SOC group, p=0.004). 

Reviewer 3: ```suggestion
The most common adverse outcome in the SOC+HCQ group was diarrhea (10% vs. 0% in the SOC group, p=0.004). 
```

The study had a few limitations. 

> One of the limitations of the study was that the cohort mostly consisted of patients with mild/moderate disease symptoms, and the average patient in the analysis was 46.

Reviewer 3: It's about the age I guess?
```suggestion
One of the limitations of the study was that the cohort mostly consisted of patients with mild to moderate symptoms, and the average age of analyzed patients was 46.
```

Reviewer 2: @matfax Right now this text mainly addresses the age of patients in this study, but there is something to be said about the fact that they're analyzing mild to moderate cases whereas a lot of other studies are focusing on quite severe cases! However, with HCQ, it seems like mild to moderate cases are the correct population to be analyzing, because the side effects can be quite severe. I'm not sure whether we should address this possible limitation further.

Reviewer 3: I agree. Treatment during the early phase of the infection isn't as fatal (regarding side effects) and can be stopped in time. During the late phase, the virus infection could enhance the side effects.

Moreover, the efficacy of virostatic drugs inversely correlates with the viral load. But there isn't sufficient proof of the primary mechanism of HCQ in COVID-19. It's possible that it has a primary virostatic effect, but it's also possible that it primarily improves the physiological response to accelerate virus clearance.

> This may not be representative of COVID-19 patients broadly because older age groups are known to be at higher risk.

Reviewer 3: Unclear antecedent
```suggestion
This average may not be representative for COVID-19 patients because older age groups are known to be at higher risk.
```

> This may not be representative of COVID-19 patients broadly because older age groups are known to be at higher risk.

Reviewer 2: ```suggestion
Therefore, this sample may not be representative of COVID-19 patients broadly because older age groups are known to be at higher risk.
```
Building on @matfax suggestion

Although the authors claimed that HCQ could have beneficial effects in hindering disease progression, multiorgan failure, and death in COVID-19, this finding cannot be extrapolated to older patients or severe cases based on the evidence they present.
Indeed, a larger sample needs to be tested to validate whether lymphocyte counts do increase in patients treated with SOC+HCQ; although the result in the present analysis appeared promising, it lacked statistical significance.

> Additionally, in this study, SOC included the use of antivirals (Lopinavir-Ritonavir, Arbidol, Oseltamivir, Virazole, Entecavir, Ganciclovir and/or Interferon-alpha), which appeared to introduce confounding effects. 

Reviewer 3: The drug is called interferon alfa
```suggestion
Additionally, in this study, SOC included the use of antivirals (Lopinavir-Ritonavir, Arbidol, Oseltamivir, Virazole, Entecavir, Ganciclovir, and Interferon alfa), which appeared to introduce confounding effects. 
```
What are "confusing effects"? Can you give an example? Is it confusing to understand the results or were the effects confusing to understand for the researchers?

Reviewer 2: > The drug is called interferon alfa
> 
> What are "confusing effects"? Can you give an example? Is it confusing to understand the results or were the effects confusing to understand for the researchers?

```suggestion
Additionally, in this study, SOC included the use of antivirals (Lopinavir-Ritonavir, Arbidol, Oseltamivir, Virazole, Entecavir, Ganciclovir and/or Interferon-alpha).
The administration of antivirals was found during _post hoc_ analysis to introduce confounding effects to the analysis of symptom alleviation. 
```
Suggestion to improve text based on @matfax's comment

Thus, to better assess the effects of HCQ, an alternate study design needs to be outlined where SOC does not involve the use of anti-virals.

> In this trial, the samples used to test for presence of the SARS-CoV-2 virus were collected from the upper respiratory tract, and the authors indicated that the use of upper respiratory samples may have introduced false negatives; thus, biomarkers for the disease are needed. 

Reviewer 3: ```suggestion
In this trial, the samples used to test for the presence of the SARS-CoV-2 virus were collected from the upper respiratory tract and the authors indicated that the use of upper respiratory samples might have introduced false negatives.
```
What testing method did they use in particular? If it was some kind of PCR testing, what's the explanation that PCR from upper respiratory collection is prone to false negativeness?

Reviewer 2: They used RT-PCR. The authors state the use of upper respiratory samples as a limitation, which seems like it could be a valid concern (e.g., [this paper](https://jamanetwork.com/journals/jama/article-abstract/2762997)) because that study found that it is easiest to detect the virus in lower respiratory tract samples. They didn't have LR samples because the patients were not severe enough to warrant collection.

Perhaps we could add that citation?
```suggestion
In this trial, the samples used to test for presence of the SARS-CoV-2 virus were collected from the upper respiratory tract, and the authors indicated that the use of upper respiratory samples may have introduced false negatives (e.g., [@doi:10.1001/jama.2020.3786]); thus, the identification of biomarkers that can be collected non-invasively would be valuable to studies such as this one.
```

(I'm answering some of these questions because some of the concerns you raise are about suggestions I made on @soumitagh's text!)

> Future studies to unravel serum, urinary, and bronchoalveolar lavage (BAL) markers for the disease would be valuable.

Reviewer 3: Is this information particular to this study or is it relevant in general? It's might better be discussed in the diagnostics chapter. I think it's more useful to find consensus there about what tests and what biomarkers to study.

Reviewer 2: If you accept my suggestion on the line above, @soumitagh, you may be able to strike this?

Author: @rando2 : no i am not able to strike this.

Another limitation of the study that the authors acknowledge was that the HCQ treatment began, on average, at a 16-day delay from the symptom onset.
A quicker treatment regimen with HCQ needs to be investigated to elucidate how the drug affects early disease management.
Overall, the study provides promising data, although all of the findings still need to be validated in independent population cohorts. 

> Their safety analysis indicated that the adverse effects of administering HCQ to mild and moderate COVID-19 cases were manageable; further investigation is needed to confirm whether the drug indeed drives improvement in symptoms and and a reduction in the inflammatory response.

Reviewer 3: ```suggestion
Their safety analysis indicated that the adverse effects of administering HCQ to mild and moderate COVID-19 cases were manageable. Further investigation is necessary to confirm whether the drug indeed ameliorates symptoms and reduces inflammatory response.
```

Indeed, a larger sample needs to be tested to validate whether lymphocyte counts do increase in patients treated with SOC+HCQ; although the result in the present analysis appeared promising, it lacked statistical significance.
Additionally, in this study, SOC included the use of antivirals (Lopinavir-Ritonavir, Arbidol, Oseltamivir, Virazole, Entecavir, Ganciclovir, and Interferon alfa), which appeared to introduce confounding effects. 
Thus, to better assess the effects of HCQ, an alternate study design needs to be outlined where SOC does not involve the use of anti-virals.
In this trial, the samples used to test for presence of the SARS-CoV-2 virus were collected from the upper respiratory tract, and the authors indicated that the use of upper respiratory samples may have introduced false negatives (e.g., [@doi:10.1001/jama.2020.3786]); thus, the identification of biomarkers that can be collected non-invasively would be valuable to studies such as this one.

> Future studies to unravel serum, urinary, and bronchoalveolar lavage (BAL) markers for the disease would be valuable.

Reviewer 2: ```suggestion
```
